cibuild.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+set -e
+
+usage()
+{
+    echo "Options"
+    echo "  --system-msbuild-path <path>   Path to the mono msbuild build used to build msbuild" 
+    echo "  --os <os>                      OS to run (Linux / Darwin)"
```

Comment:
This can be detected by default using uname.  Check out the run-test.sh script in CoreFx for a good example.